Intègre les statuts et règlement intérieur au format Markdown

MattiSG opened the pull request:
Pour faciliter la lecture, le copier-coller, et en prévision d'une collaboration pour les mettre à jour, cette PR apporte une version en texte brut des documents fondateurs de l'association.

MattiSG committed:
Remplace les statuts au format PDF par du Markdown

MattiSG committed:
Remplace le règlement PDF par du format Markdown

MattiSG committed:
Ajoute les titres des pages dans le titre du doc

Les titres sont inférés par Jekyll à partir du premier <h1>, pas besoin de métadonnées spécifiques  :)

davidbgk commented on _pages/statuts.md:
```diff
@@ -0,0 +1,84 @@
+
+## Article 1er
+
+Il est fondé entre les adhérents aux présents statuts une association régie par la loi du 1^er ^juillet 1901 et le décret du 16 août 1901, ayant pour titre :
```
Wrong syntax for `1^er^` (which is not rendered by the markdown converter?)

davidbgk commented on _pages/statuts.md:
```diff
@@ -0,0 +1,84 @@
+
+## Article 3: "Siège"
+
+Le siège social est fixé à "La Cantine", 151 rue Montmartre, Passage des Panoramas, 12 Galerie Montmartre, 75002 Paris.
```
Should we update that address too?

MattiSG replied:
Je ne pense pas, c'est un document historique. Ceci dit, vu qu'il a déjà été amendé en 2012 pour changement d'adresse, je me demande si il ne l'a pas été à nouveau suite au dernier changement d'adresse.

@julienporot est-ce que le dernier envoi à la préfecture concernait aussi un amendement des statuts pour l'adresse ?

MattiSG replied:
Je confirme qu'il y avait aussi un amendement des statuts pour l'adresse.

davidbgk commented on _pages/index.md:
```diff
@@ -42,5 +42,5 @@ Le bureau de l'association est actuellement composé de :
 
-- [Statuts](/statuts.pdf)
-- [Réglement intérieur](/reglement.pdf)
+- [Statuts](/statuts)
```
I wonder if we should keep pdf versions too (auto-generation might be tedious, not sure with jekyll).

MattiSG replied:
Quel est l'intérêt d'une version PDF ? Si quelqu'un en a besoin il peut la générer au besoin depuis la version HTML, non ?

davidbgk replied:
Genre pour garder l'original de ceux envoyés à la pref ou des choses comme ça. Mais ce n'est peut-être pas le bon endroit pour stocker ça.

MattiSG commented:
Was closed accidentally due to target branch deletion. Reopening.

MattiSG committed:
Corrige la syntaxe markdown exposant

Utilise un caractère UTF-8 plutôt que <sup> car le support pour <sup> dépend du compilateur Markdown utilisé
